deployments/engine/docker-compose/config/master.toml

```diff
@@ -17,3 +17,4 @@ level = "debug"
 reset-interval = "2s"
 initial-interval = "1s"
 max-interval = "15s"
+max-try-time = 100
```

Comment:
How long the total retry time is？Can we reduce `max-try-time` appropriately?

Reply:
This is used in test to prevent job from termination. In product environment we use a smaller value `8`